### ISSUE TITLE
How to obtain the 86.9% mIoU in validation set?

### ISSUE THREAD

**Issue by John1231983** [DrSleep/tensorflow-deeplab-resnet#147]
Thanks for the great project. I am using python 2.7 and tensorflow 1.2. I trained the code with `deeplab_resnet_init.ckpt` after 20k iterations:

`python train.py --random-mirror --random-scale`
Loss:
```
step 19999 	 loss = 1.358, (1.004 sec/step)
The checkpoint has been created.
step 20000 	 loss = 1.456, (8.148 sec/step)
```

My Mean IoU is **75.1%** in single scale and **76.3%** in multiple scale case in the validation dataset. I tried using both trand+val dataset as for training but it only got 83% that lower than your report as 86.9% mIoU (as trainval was likely to be used for final training). Could you give me some tips to achieve your performance? Thanks

**Comment by DrSleep** [comment]
Refer to train_msc

**Comment by John1231983** [comment]
Thanks. But in the README, you said that "The single-scale model shows 86.9% mIoU...", it means it trained in single-scale as `python train.py --random-mirror --random-scale`. Am I right?

**Comment by DrSleep** [comment]
It also says 'converted pre-trained model'; pre-trained model, in this
case, has been trained with multi-scale.
